generator/src/test/java/com/linkedin/pegasus/generator/TestDataSchemaParser.java

```diff
@@ -172,6 +172,54 @@ public void testSchemaFilesInExtensionPathInJar(String[] files, String[] expecte
     }
   }
 
+  @DataProvider(name = "PegasusSchemasWithExtensionResolver")
+  private Object[][] createPegasusSchemasWithExtensionResolver()
+  {
+    return new Object[][]
+        {
+            {
+                new String[]{
+                    "pegasus/Foo.pdl",
+                    "pegasus/Bar.pdl",
+                    "pegasus/Fuzz.pdsc"
+                },
+                new String[]{}
+            },
+        };
+  }
+
+  @Test(dataProvider = "PegasusSchemasWithExtensionResolver")
+  public void testBaseSchemaFilesInExtensionResolverInJar(String[] files, String[] expected) throws Exception
```

Comment:
I don't think this test is covering the change made in the code.
Can you add a test case that also has extension schemas in the jar?
And a case where the extension schema includes a schema in /pegasus/ directory.

Reply:
Karthik, The scenario you described is already in the same test class.  For extension schema project

There were 2 test scenario here
1.  extensions and pegasus in jar
2. extensions and pegasus in path

The one I added here is to guarantee we dont parse pegasus schema in the jar
Which is a conclusion after viewing your comment and discuss with the team

Reply:
In that case, can we merge this testcase with the dataprovider above? No need to keep them separate

Reply:
Yes, merged the tests, resolving this comment for now.